test/functional/filters/apiaudit/vrl/comaptibility_test.go

```diff
@@ -0,0 +1,57 @@
+package apiaudit
```

Comment:
vector only?

Reply:
The VRL tests don't use the collector, they start vector for themselves. We could make them vector only though, I guess they're running twice now...